pages/premium/saved-posts/[domain].vue

```
@@ -227,38 +229,131 @@
+          toast.error(response.statusText, {
+            description: 'You sent too many requests in a short period of time',
+            action: {
+              label: 'Verify I am not a Bot',
+              onClick: () => window.open(config.public.apiUrl + '/status', '_blank')
+            }
+          })
+          tagResults.value = []
+          break
+
+        default:
+          const Sentry = await import('@sentry/nuxt')
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)